lisa/runner.py

```diff
@@ -33,6 +33,39 @@ def parse_testcase_filters(raw_filters: List[Any]) -> List[schema.BaseTestCaseFi
     return filters
 
 
+def print_results(
+    test_results: List[Union[TestResult, TestResultMessage]],
```

Comment:
Good to see it! so the original method in Runner can be removed, this method can be shared by runner and notifier.